src/datum_transformations.jl

```diff
@@ -0,0 +1,114 @@
+# Generic datum shifting function in ECEF coordinates
+
+"""
+    datum_shift_ECEF(dest_datum, source_datum)
+
+Return a transformation object which transforms ECEF points from `source_datum`
+to `dest_datum`.  The function should attempt to supply the current publicly
+accepted best estimate.
+
+Note that the best known version of a datum transformation will inherently
+improve with time (see below), so we *cannot simultaneously* guarentee that:
```

Comment:
you must have been reading an older version, I already fixed this yesterday ;-)

Thanks though!  (the spellchecker also let me know that I can't spell Britain)